google-cloud-storage/samples/storage_configure_retries.rb

```diff
@@ -0,0 +1,51 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START storage_configure_retries]
+def configure_retries bucket_name: nil, file_name: nil
+  # The ID of your GCS bucket
+  # bucket_name = "your-unique-bucket-name"
+
+  # The ID of your GCS object
+  # file_name = "your-file-name"
+
+  require "google/cloud/storage"
+
+  # Customize retry with the maximum retry attempt of 5 (default=3).
```

Comment:
You don't have to give all the default values here -- they will be in a separate section above the sample in the docs. See https://cloud.google.com/storage/docs/retry-strategy#tools

Reply:
This sample was developed using python sample as reference. And they have mentioned the default values in the comments. So should I still remove the default values ?

Reply:
I would still say remove them, they are not present for other languages such as Go and Java. I think the python sample was developed before the full section was written which includes the table above the sample.

Reply:
Got it. Removed the default value.